homeworks/iurii.stepanenko_IuriiSky/homework_5/task3.js

```diff
@@ -0,0 +1,7 @@
+const obj = { b: 'c', d: { e: 'f' } };
+function innerCopy(newObj) {
+    return Object.assign({}, newObj);
+}
```

Comment:
this is not a deep copy as objects `d` property still reference the same object

Reply:
I fix this task. I hope it`s right.